UWP doesn't like GL_BGRA

**Godot version:**
Godot Master 19 May 2018

**OS/device including version:**
UWP

**Issue description:**
Getting the following compiler error for UWP:
```
rasterizer_canvas_gles3.cpp
[ 43%] Compiling ==> drivers\gles3\rasterizer_gles3.cpp
rasterizer_gles3.cpp
drivers\gles3\rasterizer_gles3.cpp(342): error C2065: 'GL_BGRA': undeclared identifier
drivers\gles3\rasterizer_gles3.cpp(399): error C2065: 'GL_BGRA': undeclared identifier
scons: *** [drivers\gles3\rasterizer_gles3.uwp.debug.64.x64.obj] Error 2
scons: building terminated because of errors.
```

**Steps to reproduce:**
Compile for UWP :)
Managed to compile a week or two ago so must be a recent change.

--- a/drivers/gles2/rasterizer_gles2.cpp
+++ b/drivers/gles2/rasterizer_gles2.cpp
@@ -346,7 +346,7 @@ void RasterizerGLES2::set_boot_image(const Ref<Image> &p_image, const Color &p_c
 
 	if (OS::get_singleton()->is_layered_allowed()) {
 		if (OS::get_singleton()->get_window_per_pixel_transparency_enabled()) {
-#ifdef WINDOWS_ENABLED
+#if (defined WINDOWS_ENABLED) && !(defined UWP_ENABLED)
 			Size2 wndsize = OS::get_singleton()->get_layered_buffer_size();
 			uint8_t *data = OS::get_singleton()->get_layered_buffer_data();
 			if (data) {
@@ -401,7 +401,7 @@ void RasterizerGLES2::end_frame(bool p_swap_buffers) {
 
 	if (OS::get_singleton()->is_layered_allowed()) {
 		if (OS::get_singleton()->get_window_per_pixel_transparency_enabled()) {
-#ifdef WINDOWS_ENABLED
+#if (defined WINDOWS_ENABLED) && !(defined UWP_ENABLED)
 			Size2 wndsize = OS::get_singleton()->get_layered_buffer_size();
 			uint8_t *data = OS::get_singleton()->get_layered_buffer_data();
 			if (data) {

--- a/drivers/gles3/rasterizer_gles3.cpp
+++ b/drivers/gles3/rasterizer_gles3.cpp
@@ -335,7 +335,7 @@ void RasterizerGLES3::set_boot_image(const Ref<Image> &p_image, const Color &p_c
 
 	if (OS::get_singleton()->is_layered_allowed()) {
 		if (OS::get_singleton()->get_window_per_pixel_transparency_enabled()) {
-#ifdef WINDOWS_ENABLED
+#if (defined WINDOWS_ENABLED) && !(defined UWP_ENABLED)
 			Size2 wndsize = OS::get_singleton()->get_layered_buffer_size();
 			uint8_t *data = OS::get_singleton()->get_layered_buffer_data();
 			if (data) {
@@ -392,7 +392,7 @@ void RasterizerGLES3::end_frame(bool p_swap_buffers) {
 
 	if (OS::get_singleton()->is_layered_allowed()) {
 		if (OS::get_singleton()->get_window_per_pixel_transparency_enabled()) {
-#ifdef WINDOWS_ENABLED
+#if (defined WINDOWS_ENABLED) && !(defined UWP_ENABLED)
 			Size2 wndsize = OS::get_singleton()->get_layered_buffer_size();
 			uint8_t *data = OS::get_singleton()->get_layered_buffer_data();
 			if (data) {

